modules/administration/pages/auth-methods-sso.adoc

```diff
@@ -0,0 +1,209 @@
+Check your IdP documentation for instructions.
+
+You will need these endpoints:
+
+* Assertion consumer service (or ACS): an endpoint to accept SAML messages to establish a session into the Service Provider.
+The endpoint for ACS in {productname} is: https://example.com/rhn/manager/sso/acs
+* Single logout service (or SLS): an endpoint to initiate a logout request from the IdP.
+The endpoint for SLS in {productname} is: https://example.com/rhn/manager/sso/sls
+* Metadata: an endpoint to retrieve {productname} metadata for SAML.
+The endpoint for metadata in {productname} is: https://example.com/rhn/manager/sso/metadata
+
+
+[IMPORTANT]
+====
+Your IdP must have a SAML:Attribute containing the username of the IdP user domain, called ``uid``.
+The ``uid`` attribute passed in the SAML:Attribute must be created in the {productname} user base before you activate single sign-on.
+====
+
+After the authentication with the IdP using the user ``orgadmin`` is successful, you will be logged in into {productname} as the ``orgadmin`` user, provided that the ``orgadmin`` user exists in {productname}.
+
+
+
+== Enable SSO
+
+[NOTE]
+====
+Using SSO is mutually exclusive with other types of authentication: it is either enabled or disabled.
+SSO is disabled by default.
+====
+
+.Procedure: Enabling SSO
+
+. If your users do not yet exist in {productname}, create them first.
+. Edit [path]``/etc/rhn/rhn.conf`` and add this line at the end of the file:
++
+----
+java.sso = true
+----
+. Find the parameters you want to customize in [path]``/usr/share/rhn/config-defaults/rhn_java_sso.conf``.
+Insert the parameters you want to customize into [path]``/etc/rhn/rhn.conf`` and prefix them with [literal]``java.sso``.
+For example, in [path]``/usr/share/rhn/config-defaults/rhn_java_sso.conf`` find:
++
+----
+onelogin.saml2.sp.assertion_consumer_service.url = https://YOUR-PRODUCT-HOSTNAME-OR-IP/rhn/manager/sso/acs
+----
++
+To customize it, create the corresponding option in [path]``/etc/rhn/rhn.conf`` by prefixing the option name with ``java.sso.``:
++
+----
+java.sso.onelogin.saml2.sp.assertion_consumer_service.url = https://YOUR-PRODUCT-HOSTNAME-OR-IP/rhn/manager/sso/acs
+----
++
+To find all the occurrences you need to change, search in the file for the placeholders [literal]``YOUR-PRODUCT`` and [literal]``YOUR-IDP-ENTITY``.
+Every parameter comes with a brief explanation of what it is meant for.
+. Restart the spacewalk service to pick up the changes:
++
+----
+spacewalk-service restart
+----
+
+When you visit the {productname} URL, you will be redirected to the IdP for SSO where you will be requested to authenticate.
+Upon successful authentication, you will be redirected to the {productname} {webui}, logged in as the authenticated user.
+If you encounter problems with logging in using SSO, check the {productname} logs for more information.
+
+
+
+== Example SSO Implementation
```

Comment:
I would split the Keycloak part to a subpage. In the not-so-distant future, we will also document other SAML providers. One subpage per SAML provider seems more sensible that a long page with lots of information.

Reply:
Yep, I already split it into PAM and SSO, happy to split the example out again.